Uplift/Views/Cells/FacilitiesDropdownCollectionViewCell.swift

```diff
@@ -0,0 +1,168 @@
+//
+//  FacilitiesDropdownCollectionViewCell.swift
+//  
+//
+//  Created by Cameron Hamidi on 11/6/19.
+//
+
+import UIKit
+
+class FacilitiesDropdownCollectionViewCell: UICollectionViewCell {
```

Comment:
thoughts on shortening the name to just `FacilitiesDropdownCell`?

Reply:
My reasoning was that it makes it clear that it's a collection view cell versus a table view cell. However if that distinction is clear elsewhere/not needed then the name can be shortened.

Reply:
i personally think `FacilitiesDropdownCell` is fine -- we're not using table views throughout the app and most of the other cell classes omit "collectionView" from the file names as well.